applications/GeoMechanicsApplication/tests/test_thermal_element/test_micro_climate_1/ProjectParameters.json

```diff
@@ -0,0 +1,209 @@
+      "preconditioner_type"              : "ilu0"
+    },
+    "problem_domain_sub_model_part_list" : ["soil1"],
+    "processes_sub_model_part_list"      : ["start","observation_point","top_boundary","bot_boundary"],
+    "body_domain_sub_model_part_list"    : ["soil1"]
+  },
+  "output_processes" : {
+    "gid_output" : [{
+      "python_module"                    : "gid_output_process",
+      "kratos_module"                    : "KratosMultiphysics",
+      "process_name"                     : "GiDOutputProcess",
+      "Parameters" : {
+        "model_part_name"                : "PorousDomain.porous_computational_model_part",
+        "output_name"                    : "test_micro_climate_1_map",
+        "postprocess_parameters" : {
+          "result_file_configuration": {
+            "gidpost_flags" : {
+              "WriteDeformedMeshFlag"    : "WriteUndeformed",
+              "WriteConditionsFlag"      : "WriteElementsOnly",
+              "GiDPostMode"              : "GiD_PostAscii",
+              "MultiFileFlag"            : "SingleFile"
+            },
+            "file_label"                 : "step",
+            "output_control_type"        : "step",
+            "output_interval"            : 240,
+            "body_output"                : true,
+            "node_output"                : false,
+            "skin_output"                : false,
+            "plane_output"               : [],
+            "nodal_results"              : ["TEMPERATURE"],
+            "gauss_point_results"        : []
+          },
+          "point_data_configuration"     : []
+        }
+      }
+    },{
+      "python_module"                    : "gid_output_process",
+      "kratos_module"                    : "KratosMultiphysics",
+      "process_name"                     : "GiDOutputProcess",
+      "Parameters" : {
+        "model_part_name"                : "PorousDomain.observation_point",
+        "output_name"                    : "test_micro_climate_1_his",
+        "postprocess_parameters" : {
+          "result_file_configuration": {
+            "gidpost_flags" : {
+              "WriteDeformedMeshFlag"    : "WriteUndeformed",
+              "WriteConditionsFlag"      : "WriteElementsOnly",
+              "GiDPostMode"              : "GiD_PostAscii",
+              "MultiFileFlag"            : "SingleFile"
+            },
+            "file_label"                 : "step",
+            "output_control_type"        : "step",
+            "output_interval"            : 240,
+            "body_output"                : true,
+            "node_output"                : false,
+            "skin_output"                : false,
+            "plane_output"               : [],
+            "nodal_results"              : ["TEMPERATURE"],
+            "gauss_point_results"        : []
+          },
+          "point_data_configuration"     : []
+        }
+      }
+    }]
+  },
+  "processes": {
```

Comment:
the processes for new variables deserve a place on our wiki.

Reply:
I added to the documentation issue, we should add this to the wiki